src/utils/helpers/createRoleOptions.ts

```diff
@@ -0,0 +1,11 @@
+import lodash from "lodash";
```

Comment:
lodash is pretty bit

https://blog.battlefy.com/tree-shaking-lodash-with-vite


if you're using only one function I suggest just implement this function (or copy it from lodash)